src/modules/map/component.ts

```diff
@@ -123,7 +123,7 @@ export async function createMapComponent(components: {
         tokens.resolve(addTokens(result.parcels, result.estates, {}))
         ready = true
         await sleep(refreshInterval)
-        await poll()
+        poll()
         events.emit(MapEvents.READY, result)
```

Comment:
can we move this one above the sleep? so the event is dispatched as soon as is ready